packages/shared/styles/components/SfNotification.scss

```diff
@@ -0,0 +1,93 @@
+@import '../variables';
+$notification__message-padding-left: 0.625rem !default;
+$notification--info-color: $c-dark-primary !default;
+$notification--warning-color: #d12727 !default;
+$notification--alert-color: #0468db !default;
+$notification--success-color: #128830 !default;
+$notification-mobile-padding: $spacer-big !default;
+$notification-background-size: cover !default;
+$notification-background-position: bottom left !default;
+$notification__title-font-size: 2rem !default;
+.sf-notification {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    box-sizing: border-box;
+    border-radius: 15px;
+    min-height: 9rem;
+    padding: $notification-mobile-padding;
+    background-size: $notification-background-size;
```

Comment:
background-size and position properties are redundant here since the background is plain color

Reply:
deleted